<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_st">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>2017-01-01</string>
   </property>
   <property name="placeholderText">
    <string>yyyy/mm/dd</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Data Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>End Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Technical Analysis Tools</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Currency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_dp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_currency">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>BTC-USD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rsi">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>RSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ma">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Moving Average</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Start Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ed">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>2017-04-30</string>
   </property>
   <property name="placeholderText">
    <string>yyyy/mm/dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Input Parameters</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ema">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Exponential</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_dp2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Data Points 2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Select for Values</string>
   </property>
   <item>
    <property name="text">
     <string>Select for Values</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simple Moving Average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exponential Moving Average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Resistance Strength Index</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_cc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Candlestick Chart</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Click below for values for graph display </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../TradeApp-TA/help.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../TradeApp-TA/taPagelogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_view">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
    </font>
   </property>
   <property name="text">
    <string>Table View</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
